up-streamer/tests/usubscription_bad_data.rs

```diff
@@ -0,0 +1,51 @@
+/********************************************************************************
```

Comment:
Recommend calling this file just ususbcription.rs and then we can put all usubscription-related tests in here.

Reply:
done